tests/test_init.py

```diff
@@ -44,6 +44,10 @@ def test_systemd_no_udev_present(self, auto_container):
     def test_systemd_boottime(self, auto_container):
         """Ensure the container startup time is below 5 seconds"""
 
+        # Temporary workaround: While using emulated workers, the startup time test doesn't make sense.'
+        if auto_container.connection.system_info.arch == "ppc64le":
+            return
```

Comment:
Please use `pytest.skip()` with a reason why here
Also, please add a giant `FIXME` somewhere

Reply:
Done.

Reply:
Can this FIXME have an issue so we dont forget about it?

Reply:
https://github.com/SUSE/BCI-tests/issues/647